themes/question/bootstrap_dropdown/survey/questions/answer/list_dropdown/answer.twig

```diff
@@ -51,15 +51,19 @@
         $(function () {
             var selectPicker = $('#answer{{ name }}').selectpicker();
 
-            // Custom js to make width setting work with this beta version:
+            {#
+                Custom js to make width setting work with this beta version.
+                The bootstrap-select's auto-width feature relies on a handler set for 'loaded.bs.select' by setWidth(),
+                which is called after the event is triggered, so the auto-width's handler is never executed.
+                This is a workaround to make the auto-width work.
+            #}
             {% if width_entry != '' %}
-            selectPicker.on('loaded.bs.select', function () {
-                selectPicker.selectpicker('open');
-                var dropdown = selectPicker.parent();
-                var width = dropdown.find('.dropdown-menu.show').width();
-                dropdown.css('width', width);
-                selectPicker.selectpicker('toggle');
-                $('.bootstrap-select .btn.dropdown-toggle').blur();
+            selectPicker.on('loaded.bs.select.firstload', function () {
+                selectPicker.off('loaded.bs.select.firstload');
```

Comment:
Add a data done ? I dislike remove ALL action : this disable js workaround if needed

Reply:
This is a custom event, created and used only for this workaround purpose. It will not deactivate anything else.
Adding a data-workaround-done attribute seems too much to me.

Reply:
OK great ! I don't know it was a custom event firstload didn't exist in bs.select ?

Reply:
Nope. Created for this matter